### DISCUSSION TITLE
Allow comments in multiline text node?

### DISCUSSION THREAD

**Discussion by judah-caruso** [WASasquatch/was-node-suite-comfyui#29] [Ideas]
Was going to submit a PR but wanted to ask in case you didn't like the idea. How do you feel about allowing comments in the multiline text node? I currently just wrap things I want "commented out" in `< >`, which works, but might make something break down the line.

Example prompt:
```
night,
# star filled sky,
large planet in the distance
````
turns into
```
night,
large planet in the distance
```

I'm not sure how much whitespace affects the prompt, so maybe something like C's block comments `/* */` would work better?

**Comment by WASasquatch** [comment]
Could just use something like `if the_line.strip().startswith('#')` and if so, omit it from the final output.

**Comment by WASasquatch** [comment]
Alright, I added comments (this update also considered Load Text File). 

![image](https://user-images.githubusercontent.com/1151589/229959281-d06ee7a8-32bf-4e49-b483-5509d2cf122e.png)
